src/System.Diagnostics.TraceSource/tests/TraceListenerClassTests.cs

```diff
@@ -49,23 +52,27 @@ public void FilterTest()
             Assert.NotNull(listener.Filter);
         }
 
-        [Fact]
-        public void TraceOutputOptionsTest()
+        [Theory]
+        [InlineData(TraceOptions.Callstack)]
```

Comment:
I'm guessing these tests ought to pass on .NET Framework as well? This test lib doesn't have such a configuration, but perhaps worth temporarily adding one in order to check they do?